src/adjoint.jl

```diff
@@ -78,7 +79,7 @@ function gradm(ex, mut = false, keepthunks = false)
         back(::Nothing) = nothing
         back(Δ::AbstractZero) = $gradtuplekw(ntuple(_ -> Δ, $(length(args))))
       end)
```

Comment:
I doubt this is why tests fail, but I forgot the unthunk in the KW path:
```suggestion
      end)
      $maybe_unthunk
```